crates/iroha/tests/triggers/time_trigger.rs

```diff
@@ -131,7 +131,7 @@ fn pre_commit_trigger_should_be_executed() -> Result<()> {
         test_client.submit(sample_isi)?;
 
         // Waiting for empty block to be committed
-        std::thread::sleep(network.pipeline_time());
+        std::thread::sleep(2*network.pipeline_time());
```

Comment:
Should there be any functionality to wait for an empty block?

Reply:
In #5380 I refined block hooks in the test network and rewrote all such places into something like

```rs
// accounts for empty blocks too
network.ensure_blocks_with(|height| height.total == 2).await; 
```